ctags/main/read.c

```diff
@@ -555,15 +554,15 @@ extern char *readLine (vString *const vLine, MIO *const mio)
 		do
 		{
 			char *const pLastChar = vStringValue (vLine) + vStringSize (vLine) -2;
-			MIOPos startOfLine;
+			long startOfLine;
 
-			mio_getpos (mio, &startOfLine);
+			startOfLine = mio_tell(fp);
```

Comment:
When I was adding MIO to uctags I found it easier to replace FILE with MIO and just follow the errors and fix them on the way (my way) than checking how it's done in Geany. So there are minor differences like that.